### ISSUE TITLE
Proposal: Make the getter for XamlCompositionBrushBase.CompositionBrush public

### ISSUE THREAD

**Issue by seanocali** [microsoft/microsoft-ui-xaml#3667]
# Proposal: [Make the getter for XamlCompositionBrushBase.CompositionBrush public] 

## Summary
I have custom controls that use CompositionBrushes internally but have XAML Brush dependency properties. I have written XAML Brush to CompositionBrush translations for solid and linear gradient, but not for acrylic or anything else that comes along.

## Rationale
I can't think of any dangers to making access to the getter public.

**Comment by codendone** [comment]
I agree there isn't a strong need to have the getter be protected. Having a public getter might suggest more flexibility than the XamlCompositionBrushBase subclass might support, though. For example:
* It is usually best practice to Close() a CompositionBrush when it is no longer needed, so anyone trying to directly use the CompositoinBrush will be broken if the XCBB calls Close() (such as because it decided to switch to a different CompositionBrush).
* Complex XCBB classes like AcrylicBrush update the CompositionBrush based on various events they monitor (like app theme, low power, and resolution scale changes), and those aspects would also be lost if someone tries to use the CompositionBrush without its associated XCBB.

A public getter could still safely be used to check if the CompositionBrush has been set, or for a test to check if the right type and properties got set, though.

**Comment by seanocali** [comment]
Good points. Plus if someone disposes the CompositionBrush before the XCBB is done with it I imagine that could be problematic. Maybe there could be a method to clone it? Something like:

`Compositor.CreateBrushFromXaml(XamlCompositionBrushBase xamlBrush)`

**Comment by codendone** [comment]
Cloning would solve the Close() issue, but not the "Complex XCBB classes" issue.
